server/test/controllers/BlocksControllerSpec.scala

```diff
@@ -434,6 +435,23 @@ class BlocksControllerSpec extends MyAPISpec {
 
   }
 
+  "GET /blocks/rewards-summary" should {
```

Comment:
This test is ok, I'm thinking that it could be worth moving everything to the stats controller instead.